src/Compiler/Graphs/MinCut.cs

```diff
@@ -171,8 +171,8 @@ public Set<NodeType> GetSourceGroup()
                     {
                         NodeType target = graph.TargetOf(edge);
                         bool isSinkEdge = IsSinkEdge(edge);
-                        int distTarget = isSinkEdge ? 0 : distanceToSink[target];
```

Comment:
You introduced an error by removing this line.

Reply:
Thanks!